code/modules/shuttle/supply.dm

```diff
@@ -596,3 +589,135 @@ GLOBAL_LIST_EMPTY(exports_types)
 		temp = null
 
 	updateUsrDialog()
+
+//A tablet that can order requisition goodies; when it's linked to a beacon it will deliver the goods via launch pods nearby the beacon
+
+/obj/item/req_tablet
+	name = "requisition tablet"
+	desc = "A handheld tablet that allows the average user to remote order supply packages. When this is linked to a beacon, it will deliver the goods VIA launch pods. \
+			The pods will drop in a 5x5 square around the beacon and will not injure or explode, it will teleport away shortly after dropping off the contents."
+	var/obj/item/radio/beacon/linked_beacon
+	icon = 'icons/obj/items/req_tablet.dmi'
+	icon_state = "req_tablet_off"
+	var/podType = /obj/structure/closet/supplypod/centcompod //No explosion, fast landing and teleports itself out of the landing destination
+	var/temp //TGUI things
+	var/last_viewed_group = "categories"
+
+/obj/item/req_tablet/examine(mob/user)
+	. = ..()
+	if(!linked_beacon)
+		to_chat(user, "There seems to be no beacon attached to this tablet, hit a beacon with a tablet to link to it.")
+	else
+		to_chat(user, "There seems to be a beacon attached to this tablet. Should be good to go for delivery.")
+	return .
+
+/obj/item/req_tablet/afterattack(atom/A, mob/user, proximity)
```

Comment:
Shouldn't this be attack_by otherwise wont you hit them first then link it?
Actually hit it and take damage i mean

Reply:
Haven't gotten the beacon to take damage from hitting it with a tablet

Reply:
Maybe `preattack` then?